Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,13 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Parallel RBD PVC creation hangs for new BlockPool
+
+This issue is specifically present in CephCSI `v3.4.x`, used by rook `>=v1.7.1` and occurs when multiple parallel PVCs creation requests are issued on a newly created blockpool. More details at https://github.com/rook/rook/issues/8696. Follow the steps below to workaround the issue:
```

Comment:
In what versions do we expect this to be fixed? This makes it sound like the issue will be in any version of Rook greater than 1.7.1. The ceph version also is important for this issue, right?

Reply:
The issue is in librbd (i.e. client side), so the ceph version used by ceph-csi is the only thing that matters.  The ceph version deployed by rook (monitors, OSDs, etc) is irrelevant.